styles/atom-import-cost.less

```diff
@@ -7,5 +7,5 @@
   opacity: 0.5;
   margin-left: 10px;
   position: relative;
-  top: -(@component-line-height - (@component-padding/2));
+  top: -@component-line-height;
```

Comment:
it looks pretty wells centered for me.
![screen shot 2017-08-10 at 10 38 35 am](https://user-images.githubusercontent.com/245300/29183601-2265dcc2-7db8-11e7-83f2-e9f674b4769d.png)